keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -148,19 +210,35 @@ because work lists can be implemented in a startup script using the completion
 index as building block.
 * The semantics of an indexed Job are similar to a StatefulSet, in the sense
 that Pods have an associated index.
-However, the APIs have a major difference: a StatefulSet doesn't have completion
-semantics, as opposed to Jobs.
+However, the APIs have major differences:
+  - a StatefulSet doesn't have completion semantics, as opposed to Jobs.
+  - a StatefulSet creates pods serially, whereas Job creates all Pods in
```

Comment:
perhaps mention that pod names are stable in statefulset, but they are not in indexed job.

Reply:
Done